### ISSUE TITLE
[a11y] 9.4.1.1 Correct syntax

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10737]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-4-1-1-korrekte-syntax

The check was carried out with HTTP Validator (https://validator.w3.org/nu) via the Chrome
extension "ARC Toolkit". The results were filtered with "Check for WCAG 2.0 parsing
compliance" (see: https://validator.w3.org/nu/about.html#extras).
Only WCAG-relevant errors were taken into account, i.e. the following must be guaranteed for each element
the following must be guaranteed:
a) They have complete start and end tags
b) They are correctly nested according to the specification
c) They do not contain any duplicate attributes
d) All their IDs are unique, except where the specifications allow otherwise.

### Personal

- [ ] If you open the detail view of a file and then go to the "Shares" area,
duplicate IDs appear in the code. This affects both the button "Show invited
people" button and the "Show links" button.

### Shares

- [ ] The ID "resource-table-select-all" is used several times on the page. To ensure
correct functionality, each ID on the page should be unique.

**Comment by pascalwengerter** [comment]
The three occurances of "resource-table-select-all" were from a time when accepted/declined/pending shares were present in three separate tables on the same page, which has been resolved via https://github.com/owncloud/web/pull/9718

**Comment by pascalwengerter** [comment]
duplicate IDs for status indicators have also long been fixed from what I can see in `statusIndicator.ts`

**Comment by kulmann** [comment]
Check it as well, both issues solved:
- indicator ids get the file id as suffix, making them unique
- Shares page doesn't contain 3 ResourceTables anymore, but only 1, thus the select all id is unique on the Shares page
